### PR TITLE
Chore/fix mimir integer

### PR DESCRIPTION
**Pull request by Thorian1te**
## Summary by CodeRabbit

* **Bug Fixes**
  * Improved halt detection for THORChain and MayaChain: the app now enters pause/maintenance whenever any positive halt flag is present (not just a single specific value). Status banners, warnings, and action disabling are more reliable during network halts.

* **Tests**
  * Expanded and clarified test coverage around halt detection to cover multiple scenarios and edge cases.

### CODE REVIEW

File: src/renderer/hooks/useMimirHalt.test.ts [hunk 1]
```diff
@@ -2,26 +2,34 @@ import { getMimirStatus } from './useMimirHalt'
...
+      expect(getMimirStatus(undefined, 100)).toBeFalsy()
     })
-    it('mimir = undefined + blockheight = undefined', () => {
-      expect(getMimirStatus(undefined, undefined)).toBeFalsy()
+
+    it('should return false when mimir = 0 with valid lastHeight', () => {
+      expect(getMimirStatus(0, 100)).toBeFalsy()
     })
   })
 })
```
[the remark this answers is not in the document]
Comment (Thorian1te) [reply src/renderer/hooks/useMimirHalt.test.ts:35]: mimir is never negative
